script.js

```diff
@@ -11,3 +11,273 @@ function openDropdownItem(menuElement, menuLink) {
+}
+
+const projectData = [
+  {
+    name: 'Tonic',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: './images/tonic-work.png',
+    technologies: ['html', 'css', 'javascript'],
+    pointersTitle: 'CANOPY',
+    pointersLevel: 'Back End Dev',
+    pointersYear: '2015',
+    linkToLiveVersion: '#!',
+    linkToSource: '#!',
+  },
+  {
+    name: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: './images/multi-post-stories.png',
+    pointersTitle: 'CANOPY',
+    pointersLevel: 'Back End Dev',
+    pointersYear: '2015',
+    technologies: ['html', 'css', 'javascript'],
+    linkToLiveVersion: '#!',
+    linkToSource: '#!',
+  },
+  {
+    name: 'Tonic',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: './images/tonic-work2.png',
+    pointersTitle: 'CANOPY',
+    pointersLevel: 'Back End Dev',
+    pointersYear: '2015',
+    technologies: ['html', 'css', 'javascript'],
+    linkToLiveVersion: '#!',
+    linkToSource: '#!',
+  },
+  {
+    name: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: './images/multi-post-stories2.png',
+    pointersTitle: 'CANOPY',
+    pointersLevel: 'Back End Dev',
+    pointersYear: '2015',
+    technologies: ['html', 'css', 'javascript'],
+    linkToLiveVersion: '#!',
+    linkToSource: '#!',
+  },
+];
+
+function getTechnologies(technologies) {
+  let tech = '';
+  for (let i = 0; i < technologies.length; i += 1) {
+    const currentTechnology = technologies[i];
+    tech += `<li class='tag'>${currentTechnology}</li>`;
+  }
+  return tech;
+}
+
+function generateMobileProjectData(projectList) {
+  let data = '';
+  for (let i = 0; i < projectList.length; i += 1) {
+    const currentProject = projectData[i];
+    const technologies = getTechnologies(currentProject.technologies);
+    data += `
+            <div class='work'>
+                <img src='${currentProject.featuredImage}' class='work-image' alt='multi-post-stories2'>
+                <h2 class='work-title'>${currentProject.name}</h2>
+                <div class="work-pointers">
+                  <h3>CANOPY</h3>
+                  <div class="work-pointers-bullets"></div>
+                  <p>Back End Dev</p>
+                  <div class="work-pointers-bullets"></div>
+                <p>2015</p>
+                </div>
+                <p class='work-description'>${currentProject.description}</p>
+                <ul class='tags'>
+                    ${technologies}
+                </ul>
+                <a href='#!' class='project-link' onclick='openPopUp(${i})'>
+                    See Project
+                </a>
+            </div>
+        `;
+  }
```

Comment:
- There are some little issues to be resolved on your popup modal, please fix this 🙏. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏
Your project:
-- Please pay attention to the marked section in the picture below
<img width="788" alt="tjtjjx" src="https://user-images.githubusercontent.com/73607512/216119613-8d04e06e-1206-40df-b0c9-8d29acbec656.PNG">
Design template:
<img width="434" alt="krykk" src="https://user-images.githubusercontent.com/73607512/216119671-fb7a1d92-7f64-4f0b-9052-0eb611b5719a.PNG">